docs/en_US/Blog/HPOComparison.md

```diff
@@ -0,0 +1,218 @@
+# Hyperparameter Optimization Comparison
+*Posted by Anonymous Author*
+
+Comparison of Hyperparameter Optimization algorithms on several problems.
+
```

Comment:
Please add a link from Builtint_Tuner to here.

Reply:
OK.